gce/image/scylla_gce.json

```
@@ -18,6 +18,8 @@
       "omit_external_ip": false,
       "disk_size": 20,
       "image_labels":  {
+          "branch": "{{user `branch`| clean_resource_name}}",
+          "operating_system": "{{user `operating_system`| clean_resource_name}}",
```

@hagitsegev, can we add `BUILD_ID` to the labels too (with name `build-id`)? To have it same as for AMIs.

```suggestion
          "operating_system": "{{user `operating_system`| clean_resource_name}}",
          "build-id": "{{user `scylla_build_id`| clean_resource_name}}",          
```

---

Yes.
The whole code was changed, as scripts were "merged" on https://github.com/scylladb/scylla-machine-image/commit/a7eac3d48bd3791d6f4cb6b24648e287248bbfb1
So I'll do this for all.

---

@hagitsegev thanks, but it looks like `build-id` is missed in the `image_labels` dict. Can you check?

---

@enaydanov On which build did you check it?